content/tr/abstraction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Soyutlama (Abstraction)
+status: Completed
+category: Nitelik
+tags: ["fundamental", "", ""]
+---
+
+Bilişimde soyutlama, bir [hizmet](https://glossary.cncf.io/service/) kullanıcısından (bu bir program veya insan olabilir) ayrıntıları gizleyip, bir sistemi daha genelleyici ve kolay anlaşılır hale getiren bir temsildir.
```

Comment:
```suggestion
Bilişimde soyutlama, bir hizmet kullanıcısından (bu bir program veya insan olabilir) ayrıntıları gizleyip, bir sistemi daha genelleyici ve kolay anlaşılır hale getiren bir temsildir.
```

We removed links to non-localized pages in other documents. Let's go with that approach and add a note in the spreadsheet in `services.md` column, that there is a link to it from `abstraction.md`.